dlt/pipeline/__init__.py

```diff
@@ -21,6 +22,7 @@ def pipeline(
     pipelines_dir: str = None,
     pipeline_salt: TSecretValue = None,
     destination: TDestinationReferenceArg = None,
+    loader_file_format: TLoaderFileFormat = None,
```

Comment:
it's in the run function now of course, but I'm not quite sure where to update this since we don't have a proper reference at this point (or so it seems). I think a page with all available config options would be cool.

Reply:
(for me a reference is a place where I can find all the parts that a lib has to offer, even the ones that are not mentioned in more prosaic docs, if that makes sense)